lib/Worker.js

```diff
@@ -20,6 +22,34 @@ module.exports = function(God) {
     return proc ? proc : null;
   };
 
+  var registerCron = function(proc_key) {
+    var proc = _getProcessById(proc_key.pm2_env.pm_id);
+
+    if (!proc ||
+        !proc.pm2_env ||
+        proc.pm2_env.pm_id === undefined ||
+        !proc.pm2_env.cron_restart ||
+        God.CronJobs[proc.pm2_env.pm_id] !== undefined)
+      return;
+
+    console.log('[PM2][WORKER] Registering a cron job on:', proc.pm2_env.pm_id);
+
+    God.CronJobs[proc.pm2_env.pm_id] = new cronJob({
```

Comment:
I believe a Map should be better here

Reply:
Are maps supported by node 4?

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Browser_compatibility
Since node 0.12